debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RecordMakers.java

```diff
@@ -49,12 +49,12 @@
      * 
      * @param schema the schema information about the MySQL server databases; may not be null
      * @param source the connector's source information; may not be null
-     * @param topicSelector the selector for topic names; may not be null
+     * @param ddlChangeTopicName the name of the topic to which DDL changes are written; may not be null
```

Comment:
This is unchanged from your draft https://github.com/debezium/debezium/pull/129/
Good work, past you :)